File: kmpbench/java/com/google/j2cl/benchmarking/benchmarks/AllBenchmarks.java
package com.google.j2cl.benchmarking.benchmarks;

import com.google.j2cl.benchmarking.benchmarks.simple.Fibonacci;
import com.google.j2cl.benchmarking.framework.AbstractBenchmark;
import java.util.LinkedHashMap;
import java.util.Map;

/** Provides a map of all benchmarks */
public final class AllBenchmarks {

  public static final Map<String, AbstractBenchmark> map = new LinkedHashMap<>();

  static {
    map.put("Fibonacci", new Fibonacci());
  }

  private AllBenchmarks() {}
}

File: kmpbench/kotlin/com/google/j2cl/benchmarks/AllBenchmarks.kt
package com.google.j2cl.benchmarks

import com.google.j2cl.benchmarks.octane.RayTraceBenchmark
import com.google.j2cl.benchmarks.simple.Fibonacci

object AllBenchmarks {
  val map = mapOf("fibonacci" to Fibonacci(), "raytrace" to RayTraceBenchmark())
}
